apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.stories.tsx

```diff
@@ -28,6 +28,140 @@ experience.skills?.push(skill);
 poolCandidate.user.experiences?.push(experience);
 poolCandidate.user.userSkills?.push(fakeUserSkills(1, skill)[0]);
 
+const profileSnapshot = {
```

Comment:
Could you use generated fake data, specifically for the `experiences` and `poolCandidates` keys instead so that it can be reused? We already have `fakePoolCandidates` and `fakeExperiences`, based on variables set previously in this file.

Reply:
Oh ya, I could write it nicer https://github.com/GCTC-NTGC/gc-digital-talent/pull/9865/commits/7540f18e71ad852f022f3040f6063555a752a77d